runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -203,7 +203,7 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
         /*
          * Max size for a write request.
          */
-        int maxWriteSize = Integer.MAX_VALUE;
+        int maxWriteSize = 10_000_000;
```

Comment:
Are we changing the default 25MB?

Reply:
Yes, Maithem suggested we reduce it further from 25MB. So,  I made the default to 10MB to err on the side of caution.

Reply:
But with this PR we are not checking compressed size anywhere at all, are we?

Reply:
No, we are not checking it

Reply:
Wait...now we are only checking uncompressed size, and we still want to reduce the 25MB (compressed) to 10MB (uncompressed)?